cli/tests/pcluster/config/test_update_policy.py

```diff
@@ -1295,3 +1298,116 @@ def test_condition_checker_managed_placement_group(
     assert_that(actual_top).is_equal_to(expected_result_top)
     actual_message = fail_reason_managed_placement_group(change, patch)
     assert_that(actual_message).is_equal_to(expected_message)
+
+
+@pytest.mark.parametrize(
+    "base_config, target_config, change, expected_subnet_updated, expected_fail_reason, expected_action_needed",
+    [
+        # If change includes SubnetIds and existing + new cluster configuration uses the same managed Fsx for Lustre
```

Comment:
Please add a mixed case of managed + unmanaged FSx, e.g. 
e.g.

1)
base config = managed FSx 1
target config = managed FSx 1 is removed + unmanaged FSx 2 is added

---

2)
base config = managed FSx 1
target config = managed FSx 1 is kept + unmanaged FSx 2 is added